# These configurations attempt to install missing dependencies using pip, which
# violates the network sandbox. Instead, we'll manage the dependencies in
# portage.
--- a/setup.py
+++ b/setup.py
@@ -104,7 +104,7 @@
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     dependency_links=[],
-    install_requires=read_requirements(),
+    # install_requires=read_requirements(),

     # List additional groups of dependencies here (e.g. development
     # dependencies). You can install these using the following syntax,
@@ -114,7 +114,7 @@
         'doc': read_requirements('doc'),
         'license': read_requirements('license'),
     },
-    setup_requires=['setuptools_scm<6.0'],
+    # setup_requires=['setuptools_scm<6.0'],
     use_scm_version=True,

     # If there are data files included in your packages that need to be
